src/components/application_manager/include/application_manager/application_impl.h

```diff
@@ -493,6 +537,10 @@ class ApplicationImpl : public virtual Application,
   std::string bundle_id_;
   AppFilesMap app_files_;
   std::set<mobile_apis::ButtonName::eType> subscribed_buttons_;
+  std::map<int32_t, hmi_apis::Common_ButtonName::eType>
+      pending_subscription_buttons_;
```

Comment:
@mked-luxoft May be `pending_button_subscriptions`? As well as related function names

Reply:
@AByzhynar fixed in 2b001c6014447a350d41a2bcd1e834fd2c83057a